params/default.go

```diff
@@ -21,30 +21,36 @@ func DefaultNetwork() Network {
 
 func init() {
 	// check if custom network option set
+	// format: CELESTIA_CUSTOM=<netID>:<trustedHash>:<bootstrapPeerList>
```

Comment:
I'm again confused: is this the genesis hash or a trusted hash? @Wondertan yesterday mentioned that this should not be called trustedHash bc it is in fact the genesis hash.